### PR TITLE
Allows better setting of action button keybinds

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
Allows better setting of action button keybinds, you can now bind these in the keybind menu and they persist through rounds.
reliant on #24060
TODO:
~~- [ ] Improve blob's action button UI so that things can be binded to it~~
too much stuff to refactor, will do it eventually
- [x] Clear out some of the actions that aren't properly named
- [x] Finish up the missing action descs, there's a lot of them for item actions
- [x] Test this with SQL enabled. Should work fine but better safe than sorry
- [x] Add keybinds to the new player menu, very important so they shouldn't be hidden
- [x] Make sure spells work with this when client is already present

The backend of this is ENTIRELY string based to prevent the need to dynamically create subtype objects/manage a non runtime generated associated list

## Why It's Good For The Game
QOL + generally easier to understand UI
## Images of changes
Screenshot 2024-04-19 154242

## Testing
Compiled, ran, used a few actions in game

## Changelog
:cl:
tweak: Action button binds can now be set in the keybinds menu
tweak: a few action buttons have been renamed 
tweak: many action buttons have had their descriptions expanded detailing their use
/:cl:

### CODE REVIEW

File: code/__DEFINES/preferences_defines.dm [hunk 1]
```diff
@@ -133,6 +148,9 @@
 #define TAB_ANTAG 	2
 #define TAB_GEAR 	3
 #define TAB_KEYS 	4
+    #define SUBTAB_NORMAL_KEYBINDS 0
+    #define SUBTAB_ACTION_KEYBINDS 1
```
Comment (GDNgit) [inline code/__DEFINES/preferences_defines.dm:152]:
```suggestion
	#define SUBTAB_NORMAL_KEYBINDS 0
	#define SUBTAB_ACTION_KEYBINDS 1
```

File: code/controllers/configuration/sections/general_configuration.dm [hunk 2]
```diff
@@ -93,7 +93,6 @@
 	CONFIG_LOAD_BOOL(guest_ban, data["guest_ban"])
 	CONFIG_LOAD_BOOL(allow_antag_hud, data["allow_antag_hud"])
 	CONFIG_LOAD_BOOL(restrict_antag_hud_rejoin, data["restrict_antag_hud_rejoin"])
-	CONFIG_LOAD_BOOL(respawn_enabled, data["respawn_enabled"])
```
Comment (ZephyrTFA) [inline code/controllers/configuration/sections/general_configuration.dm:96]: why are you removing this

File: code/controllers/subsystem/non_firing/SSchangelog.dm [hunk 3]
```diff
@@ -97,7 +97,7 @@ SUBSYSTEM_DEF(changelog)
 /client/verb/changes()
 	set name = "Changelog"
 	set desc = "View the changelog."
-	set category = "OOC"
+	set category = null
```
Comment (ZephyrTFA) [inline code/controllers/subsystem/non_firing/SSchangelog.dm:100]: is this intended

File: code/game/jobs/job_exp.dm [hunk 4]
```diff
@@ -32,18 +32,6 @@ GLOBAL_LIST_INIT(role_playtime_requirements, list(
...
-/client/verb/cmd_check_own_playtime()
-	set category = "Special Verbs"
-	set name = "Check my playtime"
-
-	if(!GLOB.configuration.jobs.enable_exp_tracking)
-		to_chat(src, "<span class='warning'>Playtime tracking is not enabled.</span>")
-		return
-
-	to_chat(src, "<span class='notice'>Your [EXP_TYPE_CREW] playtime is [get_exp_type(EXP_TYPE_CREW)].</span>")
-
```
Comment (ZephyrTFA) [inline code/game/jobs/job_exp.dm:46]: :thonk:

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]
you have a lot of changes in here, such as removing some admin verbs, that are either very unintended or from a bad master merge

**Comment by Scribble-Sheep** [comment]
Wait, is this gonna replace the temporary binds creatable by ctrl-shift clicking action buttons?
Idk if diving into the preferences menu every time I needed to change F from my clown recorder to something else would be ideal.

**Comment by GDNgit** [comment]
> Wait, is this gonna replace the temporary binds creatable by ctrl-shift clicking action buttons? Idk if diving into the preferences menu every time I needed to change F from my clown recorder to something else would be ideal.

you can do both

File: code/_globalvars/lists/keybindings_lists.dm [hunk 5]
```diff
@@ -21,3 +21,33 @@ GLOBAL_LIST_INIT(keybindings_groups, list(
...
+	"Demon Actions" = AKB_CATEGORY_DEMON,
+	"Terror Spider Actions" = AKB_CATEGORY_TERROR_SPIDER,
+	"Alien Actions" = AKB_CATEGORY_ALIEN,
+	"Guardian Actions" = AKB_CATEGORY_GUARDIAN,
+	"Revenant Actions" = AKB_CATEGORY_REVENANT,
+	"Lavaland Elite Actions" = AKB_CATEGORY_LAVALAND_ELITE,
+	"Abductor Actions" = AKB_CATEGORY_ABDUCTOR,
+	"Blob Actions" = AKB_CATEGORY_BLOB,
+	"Martial Arts Actions" = AKB_CATEGORY_MARTIAL_ART,
+	"Martial Arts Actions" = AKB_CATEGORY_MARTIAL_ART,
```
Comment (Henri215) [inline code/_globalvars/lists/keybindings_lists.dm:44]: Duplicated?

File: code/game/mecha/mecha_actions.dm [hunk 6]
```diff
@@ -55,6 +58,7 @@
 
 /datum/action/innate/mecha/mech_toggle_lights
 	name = "Toggle Lights"
+	desc = "Toggles your mecha's lights"
```
Comment (Henri215) [inline code/game/mecha/mecha_actions.dm:61]:
```suggestion
	desc = "Toggles your mecha's lights."
```

File: code/_globalvars/lists/keybindings_lists.dm (hunk 5 above)
Comment (DGamerL) [inline code/_globalvars/lists/keybindings_lists.dm:44]:
```suggestion
```

**Pull request closed without merging**